Topics/User_Experience/AB_Testing.md

```diff
@@ -0,0 +1,36 @@
+# A/B Testing
+
+## **Introduction:**
+A/B testing is a research method used to compare two versions of a web page or mobile app to determine which variation of the UI results in better performance. It allows individuals and companies to optimize user experience and make improvements to their application. 
+
+## **Steps/How To:**
+**Identify Changes:** Identify any potential problem areas within the UI and create a hypothesis of how to fix it <br />
+
+**Create Variation:** Create two versions of the applications UI to run the experiment on, create the new design based on the hypothesis, whether it be changing the font, colors, or re-designing the whole page. <br />
+- The original page is known as the control.
+- The altered page is known as the variation. 
+
+**Running The Experiment:** Randomly filter the traffic of the application where one group (Group A) sees the original, or controlled version, and the other group (Group B) sees the modified version. <br />
+
+**Collect Data:** Record and measure different metrics such as user session duration or other performance metrics related to the application. <br />
+
+**Analyze Results:** Based on the results, determine which version improves performance and deploy the winning variation. The experiment can be repeated to make further improvements. <br />
```

Comment:
How can you determine which version improves performance? Is there a criteria or set of rules? Do the developers themselves make the criteria, or is there a set of well-known rules out there? 

This reminds me a bit of expert evaluations in UI/UX design. I learned about them recently in CSC318! One type of expert evaluation is a heuristic evaluation — essentially, a UX expert goes through your prototype and assesses it based on a set of heuristics (Jakob Nielsen’s 10 heuristics are the most commonly used). Then, you could change your design based on the assessments. How can you determine which version improves performance? Is there a criteria or set of rules? Do the developers themselves make the criteria, or is there a set of well-known rules out there?

Reply:
I will add some more detail to the steps to give readers an idea on how these rules are made. But it's mostly general because it is dependent on the web page and their own personal goals.